### PR TITLE
fix: send money logic with new bottom sheet nav

### PR DESCRIPTION
**Pull request by PratyushSingh07**
Jira task: https://mifosforge.jira.com/browse/MW-45

Old Build: 

https://github.com/openMF/mobile-wallet/assets/90026952/22c4bfeb-4616-4f8a-88e6-f724df447e72

New Build: 

https://github.com/openMF/mobile-wallet/assets/90026952/3155661f-ecb7-47f5-beb6-d04576f9fa9f

### CODE REVIEW

File: mifospay/src/main/java/org/mifospay/home/screens/FinanceScreen.kt [hunk 1]
```diff
@@ -3,25 +3,22 @@ package org.mifospay.home.screens
 import androidx.compose.foundation.ExperimentalFoundationApi
```
Comment (AdityaKumdale) [inline mifospay/src/main/java/org/mifospay/home/screens/FinanceScreen.kt]: unused import

File: mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt [hunk 2]
```diff
@@ -89,5 +94,5 @@ enum class PaymentsScreenContents {
 @Preview(showBackground = true)
```
Comment (AdityaKumdale) [inline mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt:92]: why is preview using viewmodel

File: mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt [hunk 3]
```diff
@@ -26,10 +26,18 @@ fun PaymentsRoute(
...
+        vpa = vpa,
+        showQr = showQr,
+        onNewSI = onNewSI,
+        onAccountClicked = onAccountClicked,
+        viewReceipt = viewReceipt,
+        proceedWithMakeTransferFlow = proceedWithMakeTransferFlow
+    )
 }
 
 @OptIn(ExperimentalFoundationApi::class)
```
Comment (AdityaKumdale) [inline mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt]: unused

File: mifospay/src/main/java/org/mifospay/payments/send/SendScreenRoute.kt [hunk 4]
```diff
@@ -45,6 +45,7 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.navOptions
```
Comment (AdityaKumdale) [inline mifospay/src/main/java/org/mifospay/payments/send/SendScreenRoute.kt]: unused

**Review by AdityaKumdale — CHANGES_REQUESTED** [review]

File: mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt (hunk 2 above)
Comment (PratyushSingh07) [reply mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt:92]: i dont think it is @AdityaKumdale

Comment (AdityaKumdale) [reply mifospay/src/main/java/org/mifospay/home/screens/PaymentsScreen.kt:92]: Is the preview working @PratyushSingh07

**Pull request merged**